_posts/2016-06-10-smart-contract-security.md

```diff
@@ -26,16 +30,24 @@ contract auction {
     highestBidder = msg.sender;
     highestBid = msg.value;
   }
-}</code></pre>
-Because of the maximal stack depth of 1024 the new bidder can always increase the stack size to 1023 and then call <code>bid()</code> which will cause the <code>send(highestBid)</code> call to silently fail (i.e. the previous bidder will not receive the refund), but the new bidder will still be highest bidder. One way to check whether <code>send</code> was successful is to check its return value:
-<pre><code>/// THIS IS STILL A NEGATIVE EXAMPLE! DO NOT USE!
+}
+```
+
+Because of the maximal stack depth of 1024 the new bidder can always increase the stack size to 1023 and then call ``bid()`` which will cause the ``send(highestBid)`` call to silently fail (i.e. the previous bidder will not receive the refund), but the new bidder will still be highest bidder. One way to check whether ``send`` was successful is to check its return value:
+
+```solidity
+/// THIS IS STILL A NEGATIVE EXAMPLE! DO NOT USE!
 if (highestBidder != 0)
   if (!highestBidder.send(highestBid))
-    throw;</code>
+    throw;
+```
+
+The ``throw`` statement causes the current call to be reverted. This is a bad idea, because the recipient, e.g. by implementing the fallback function as ``function() { throw; }`` can always force the Ether transfer to fail and this would have the effect that nobody can overbid her.
 
-The <code>throw</code> statement causes the current call to be reverted. This is a bad idea, because the recipient, e.g. by implementing the fallback function as <code>function() { throw; }</code> can always force the Ether transfer to fail and this would have the effect that nobody can overbid her.</pre>
```

Comment:
This was somewhat invalid because the `</pre>` was only here at the end of the paragaph. Should have rendered wrongly, but of course browsers try to workaround user bugs.